test/utils/validation.test.ts

```diff
@@ -0,0 +1,186 @@
+import { ObjectId } from 'mongodb'
+import {
+  Property,
+  validateObjectId,
+  validateProperties,
+} from 'utils/validation'
+
+describe('validation', () => {
+  describe('validateObjectId', () => {
+    test('valid id returns true', () => {
+      const validObjectId = new ObjectId()
+      const objectIdStr = validObjectId.toString()
+
+      expect(validateObjectId(objectIdStr)).toBe(true)
+    })
+
+    test('empty string returns false', () => {
+      expect(validateObjectId('')).toBe(false)
+    })
+
+    test('object id that gets transformed returns false', () => {
+      const invalidObjectId = 'aaaabbbbcccc'
```

Comment:
Could you change this to be a 24 character ObjectId, like the ones that we use in the database?

Reply:
will the same transformation occur if its 24 chars?

Reply:
No, but it is still possible to have an invalid 24-char ObjectId (I believe the 24-char ones are in hex). Maybe make one more test for that case? Just because we typically deal with the 24-char ones.

Reply:
added new test with invalid 24-char object ID